src/immatch/lib/triangles_vote.c

```diff
@@ -101,9 +101,13 @@ vote_triangle_matches(
             l_coord = l_tri->vertices[j];
             r_coord = r_tri->vertices[j];
             li = l_coord - left;
-            assert(li >= 0 && li < nleft);
```

Comment:
The code here was transferred 15 years ago from SVN, so I cannot find out who wrote this or why.

Reply:
I am against removing code without understanding it. assert statements have a purpose.

Reply:
@mcara , in principle, I agree. Unfortunately, I need to get back to my day job and unable to deep dive. If I had time, I would follow the logic higher up and find out why only Windows falls back to this code, while other OSes happily exit earlier. Feel free to follow that hunch.

In the meantime, since looks like this PR not going to merge any time soon, I can open another PR to fix your workflows without the Windows part.